geoalchemy2/elements.py

```diff
@@ -81,39 +85,19 @@ def __getstate__(self):
             'srid': self.srid,
             'data': str(self),
             'extended': self.extended,
-            'name': self.name,
         }
         return state
 
     def __setstate__(self, state):
-        self.__dict__.update(state)
```

Comment:
That might be the cause of the behavior change with PyPy. I am not fully sure, but that's my best intuition for the moment.